src/vdaf/mastic.rs

```diff
@@ -379,32 +391,500 @@ where
     }
 }
 
+/// Mastic prepare state.
+///
+/// State held by an aggregator between rounds of Mastic preparation. Includes intermediate
+/// state for Szk verification, the output shares currently being validated, and
```

Comment:
Wherever you refer to a struct defined in this crate, you should make it a link. Likewise below.
```suggestion
/// state for [`Szk`] verification, the output shares currently being validated, and
```